modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -462,8 +462,7 @@ object StepsTable extends Columns {
     tableState:      TableState[TableColumn],
     breakpointHover: Option[Int],
     selected:        Option[StepId],
-    scrollCount:     Int,
-    scrollWhileRun:  Boolean
```

Comment:
Good riddance, this was overcomplicated and not that useful

Reply:
I probably would have implemented it too. It was one of first things that came to mind when you explained the scroll mechanism.